src/api/tokens/mod.rs

```diff
@@ -0,0 +1,38 @@
+// This file is part of rust-web/twig
+//
+// For the copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+/*!
+Tokens, received from lexer output.
+*/
```

Comment:
They tried to remove them without success several times - I think at this point it is unlikely to happen :D